requirements.txt

```diff
@@ -1,6 +1,6 @@
 # requirements-dev
 pre-commit==2.13.0
-black
+# black
```

Comment:
We need black right?

Reply:
it is listed twice so commented once because it was giving issues

Reply:
Oohh then lets just remove once

Reply:
done